src/en/guide/theWebLayer/interceptors/interceptorOrdering.adoc

```diff
@@ -27,11 +27,11 @@ int order = HIGHEST_PRECEDENCE + 50
 int order = LOWEST_PRECEDENCE - 50
 ----
 
-To find out the computed order of interceptors you can add a debug logger to `logback.groovy` as follows:
+To find out the computed order of interceptors you can add a debug logger to `grails-app/conf/logback-spring.xml` as follows:
```

Comment:
`grails-app/conf` is not the only location for a `logback-spring.xml` file. For instance, you can also place it in `src/*/resources`. It might be better not to specify a particular path.

Reply:
Should we document all the locations as well in the introductory chapter?

Reply:
> Should we document all the locations as well in the introductory chapter?

I'd say, let's leave that task for another day.